plotsquared/installation/installation.md

```diff
@@ -0,0 +1,156 @@
+5. Reboot the server and PlotSquared will generate all necessary files.
+
+{% hint style="info" %}
+Version 6.0.0 and higher require Java 17 to run.
+{% endhint %}
+
+## Database Configuration
+
+This section explains how to configure PlotSquared database access.
+
+PlotSquared allows the database types "MYSQL" and "SQLite". If you don't know the difference of both systems you find a short compare at https://dzone.com/articles/sqlite-vs-mysql[here].
+
+{% hint style="tip" %}
+You can convert to MySQL/SQLite at any time using the `/plot database` command.
+{% endhint %}
+
+Please remember that you can use MySQL or SQLite, but not both at the same time.
+
+### Database: SQLite
+
+If you don't have a MySQL database, storage will be automatically set to "SQLite" and you may skip this step.
+(The config file is located at `/plugins/PlotSquared/config/storage.yml`.)
+
+### Database: MySQL
+
+*Enabling MySQL also supports other storage types like MariaDB:*
+
+* Navigate to `/plugins/PlotSquared/config/storage.yml`
+* Configure the credentials for your MySQL database.
+
+## Plot Setup
+
+You can now create the plotworld or setup single plots in an existing world. To do the latter, head over [here](../customization/single-plot-area.md).
+
+### Setup for a default world
+
+If you don't need a vanilla world you can set up the default world as a plotworld.
+
+1. (Stop your server and) delete the vanilla worlds `world`, `world_nether` and `world_the_end` like selected below:
+
+![image](https://i.imgur.com/6kAMx34.png)
+
+2. Then, open the file called `server.properties`; this is located in your server's root folder.
+
+- Find the following line:
+
+```text
+level-name=world
+```
+
+- Replace `world` with the name of your plot world. In this example case, we would need to name it to `plotworld`.
+So it looks like this:
+
+```text
+level-name=plotworld
+```
+
+3. Edit the bukkit.yml:
+ ** Open the file called `bukkit.yml` (it's also stored in your server's root folder). Now we need to tell the server which generator based on the world should be generated, *otherwise your world generation will be messed up*.
+
+The value we are going to edit does not exist in the file yet, so we need to add this at the bottom of the file:
+
+```yaml
+worlds:
+plotworld:
+  generator: PlotSquared
+```
+
+4. Then you can follow the next steps.
+
+## Creation via Plot Setup
+
+You can create a plotworld using the setup wizard. Use `/plot setup` to start the wizard.  +
+Every step requires the command, e.g. `/plot setup PlotSquared` and replace "PlotSquared" with your desired value.
+When you are done, PlotSquared will teleport you to the generated plotworld.
+
+### Creation via third parties
+
+#### Creation via Hyperverse - Outdated
+
+[Hyperverse](https://www.spigotmc.org/resources/77550) will (in most cases) automatically detect your plotworld as it is being generated by PlotSquared. If it doesn't, you can use `/hv import <worldname> PlotSquared` to import it.
```

Comment:
Should be deleted, Hyperverse is no longer maintained and doesn't support modern MC.